contracts/governance/Staking/Checkpoints.sol

```diff
@@ -39,6 +39,63 @@ contract Checkpoints is StakingStorage, SafeMath96 {
 
 	event ContractCodeHashRemoved(bytes32 hash);
 
+	/**
+	 * @notice Increases the user's vesting stake for a giving lock date and writes a checkpoint.
+	 * @param account The user address.
+	 * @param lockedTS The lock date.
+	 * @param value The value to add to the staked balance.
+	 * */
+	function _increaseUserVestingStake(
```

Comment:
there is supposed to be just a total vested stake, not a vested stake per user. a user is either a voluntary staker or a vesting contract, never both. 

didn't see this on the girst quick review, thought it was just weird naming of that function